### ISSUE TITLE
App Router: Plurals not working with getScopedI18n

### ISSUE THREAD

**Issue by fellipeutaka** [QuiiBz/next-international#87]
Hey!
I was trying to use the plural translations with scoped translations and I've got some issues. For some reason, it only works with `useI18n` and `getI18n`.

My dictionary:
```ts
export default {
  home: {
    "open-source": {
      "stars#one": "1 star on GitHub",
      "stars#other": "{count} stars on GitHub",
    },
  },
} as const;
```

My page component:
```tsx
import { getScopedI18n } from "~/lib/next-international/server";
import { getGitHubStars } from "~/utils/get-github-stars";

export const revalidate = 60; // 1 minute

export default async function Page() {
  const stars = await getGitHubStars();
  const t = await getScopedI18n("home");

  return (
    <main className="container grid items-center animate-in fade-in slide-in-from-bottom-8 duration-really-slow">
      <div className="flex items-center">
        <div className="h-4 w-4 border-y-8 border-l-0 border-r-8 border-solid border-muted border-y-transparent" />
        <div className="flex h-10 items-center rounded-md border border-muted bg-muted px-4 font-medium">
          {t("open-source.stars")}
        </div>
      </div>
    </main>
  );
}
```

Dependencies version:
```json
{
  "next": "13.4.12",
  "next-international": "^0.7.0",
  "typescript": "5.1.6"
}
```

Unfortunately, I couldn't find any examples of how to use plural translations with scoped translations. I would be very grateful if you could help me. By the way, thank you and congratulations for the excellent work. I've already used other internationalization libraries and this one, without a doubt, is the best of all.

**Comment by QuiiBz** [comment]
Good catch, that's indeed a bug only appearing with scoped translations. I've merged #88 which solves the issue, and it will be released soon: https://github.com/QuiiBz/next-international/issues/62#issuecomment-1659633372

**Comment by QuiiBz** [comment]
Released in [0.8.0](https://github.com/QuiiBz/next-international/releases/tag/0.8.0)!
